### ISSUE TITLE
 Home page still look ugly af

### ISSUE THREAD

**Issue by aquaswim** [aquaswim/simple-start-page#5]

**Comment by aquaswim** [comment]
use this menu, generated by chat gpt
```html
<!DOCTYPE html>
<html>
<head>
    <title>Server Services</title>
    <style>
        body {
            font-family: Arial, sans-serif;
            margin: 20px;
            background-color: #1f1f1f;
            color: #fff;
        }
        
        h1 {
            color: #fff;
            text-align: center;
        }
        
        .services {
            display: grid;
            grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
            grid-gap: 20px;
        }
        
        .service-item {
            background-color: #333;
            padding: 20px;
            border-radius: 5px;
            text-align: center;
            transition: background-color 0.3s;
            cursor: pointer;
            display: flex;
            flex-direction: column;
            align-items: center;
        }
        
        .service-item:hover {
            background-color: #555;
        }
        
        .service-icon {
            font-size: 40px;
            margin-bottom: 10px;
        }
        
        .service-heading {
            font-size: 18px;
            margin-bottom: 5px;
        }
        
        .service-subheading {
            font-size: 14px;
        }
        
        .service-link {
            text-decoration: none;
            color: #fff;
            transition: color 0.3s;
        }
        
        .service-item:hover .service-link {
            color: #ffcc00;
        }
    </style>
</head>
<body>
    <h1>Server Services</h1>
    <div class="services">
        <div class="service-item">
            <div class="service-icon">ðŸ”§</div>
            <div class="service-heading">Service 1</div>
            <div class="service-subheading">Description of Service 1</div>
            <a href="[http://service1.example.com](view-source:http://service1.example.com/)" class="service-link">Go to Service 1</a>
        </div>
        <div class="service-item">
            <div class="service-icon">ðŸ“</div>
            <div class="service-heading">Service 2</div>
            <div class="service-subheading">Description of Service 2</div>
            <a href="[http://service2.example.com](view-source:http://service2.example.com/)" class="service-link">Go to Service 2</a>
        </div>
        <div class="service-item">
            <div class="service-icon">ðŸŒ</div>
            <div class="service-heading">Service 3</div>
            <div class="service-subheading">Description of Service 3</div>
            <a href="[http://service3.example.com](view-source:http://service3.example.com/)" class="service-link">Go to Service 3</a>
        </div>
        <!-- Add more services as needed -->
    </div>
</body>
</html>
```

**Comment by aquaswim** [comment]
the chatgpt generated one is implemented in 00d0e0ecd6093cae9d9447726d073aadc8eb2bf4
